data-prepper-api/src/main/java/org/opensearch/dataprepper/model/sink/SinkLatencyMetrics.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.model.sink;
+
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+import io.micrometer.core.instrument.DistributionSummary;
+import org.opensearch.dataprepper.model.event.EventHandle;
+
+import java.time.Duration;
+import java.time.Instant;
+
+public class SinkLatencyMetrics {
+    public static final String INTERNAL_LATENCY = "internalLatency";
```

Comment:
I wonder if we should have different names here. What would a user interpret "internal" and "external" to mean?

Maybe we could follow up in a different PR before releasing 2.6.0.